tests/pb_cipher_suites.erl

```diff
@@ -66,23 +66,27 @@ confirm() ->
     ok = rpc:call(Node, riak_core_console, add_source, [["user", "127.0.0.1/32",
                                                     "password"]]),
 
-    CipherList = "AES256-SHA256:RC4-SHA",
+    CipherList = "AES128-SHA256:RC4-SHA",
     %% set a simple default cipher list, one good one a and one shitty one
-    rpc:call(Node, riak_core_security, set_ciphers,
-             [CipherList]),
+    rpc:call(Node, riak_core_security, set_ciphers, [CipherList]),
+    rpc:call(Node, application, set_env, [riak_api, honor_cipher_order, true]),
```

Comment:
Here a comment could be welcome. 
If I got it right it means: 
`If true, the Riak node uses its preference for cipher selection, despite what the client indicates.`